### ISSUE TITLE
Using a10-ansible with Ansible Tower

### ISSUE THREAD

**Issue by sebw** [a10networks/a10-ansible#49]
I'm trying to figure out how to make use of a10-ansible in a Ansible Tower (or AWX) environment.

Currently it fails with `ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.`

Can you document the installation procedure for Tower/AWX?

**Comment by mdurrant-b3** [comment]
Hi sebw - this error typically indicates that the a10-ansible module path has not been set in the ANSIBLE_LIBRARY environment variable or similar in ansible.cfg 

Can you please post the playbook you're using?

**Comment by sebw** [comment]
I managed to figure it out, that's indeed a problem of module location.

For anyone in the same situation:

Ansible Tower uses `/usr/lib/python2.7/site-packages/` by default.

I moved the folder containing Ansible modules (`a10_debug_smtp.py`, etc.) under `/opt/a10_modules`.

I moved the folder containing `axapi_http.py` under `/usr/lib/python2.7/site-packages/a10_ansible`.

Then I put an `ansible.cfg` file at the root of the playbooks repo with:

```
[defaults]
library = /opt/a10_modules
```

Now Tower can run the custom modules.

The page https://www.a10networks.com/resources/articles/installing-ansible-modules-a10-networks helped me figure out what to put where, even though the bash script is not really functional as it is.

I didn't go the `pip` way, because `pip` is not shipped by default in RHEL7.
